### ISSUE TITLE
Regex matches HTML5 <header> element

### ISSUE THREAD

**Issue by gpgekko** [gartz/html#2]
The regex you use also matches the HTML5 <header> element, which messes up my template. One way to change the regex could be to only match start and end positions for the tag.

**Comment by gartz** [comment]
Can you provide an example with your html source code that conflicts please?

**Comment by gpgekko** [comment]
Sorry it took a while, here's a snippet:

```
<article id="article">
  <header id="article-header">
    <h1>News alert</h1>
  </header>
  <section id="article-body">...</section>
</article>
```

**Comment by gartz** [comment]
I see, I couldn't reproduce here, but I believe that I understand the problem, can you patch the regexp from line 23 to:

``` js
var re = /<html(\ |>)((.|[\r\n])*)<\/html(.*)>|<head(\ |>)((.|[\r\n])*)<\/head(.*)>/i;
```

Please let me know if it solve your case, if does I will release a patch fix :)

Thanks for the help.

**Comment by gpgekko** [comment]
My mistake, the trick is to leave the linebreakes out (automated step on
our build platform to minimize html template footprint).
Op 27 jun. 2016 19:36 schreef "Gabriel Reitz Giannattasio" <
notifications@github.com>:

**Comment by gartz** [comment]
Fixed on version 1.0.0
